docs/document/content/features/transaction/use-norms/xa.en.md

```diff
@@ -13,3 +13,101 @@ weight = 2
 ## Unsupported
 
 * Recover committing and rolling back in other machines after the service is down.
+
+## XA Transaction managed by XA Statement
```

Comment:
What is the different with original XA? If same, why it appear here?

Reply:
I think the difference between `XAMANUAL` and `XA` is that users can contronl the XA transaction by themselves in `XAMANUAL` and `sharding-proxy` contronls XA transction in `XA`.

I think the motivation to add this transaction type is that users want to do the XA operations by themselves in such case like involving in two or more resources in a transacion, e.g. 
```
tx.begin();
jms.send("/queue/test", "messages"); // send to a message broker
jdbc.update("insert into db_test"); // insert into database through the proxy
tx.commit();
```

And yeah, in this scenario, users should be responsible to the transaction operations. `sharding-proxy` just bypass all of the XA related statments to the backend databases;

Reply:
> What is the different with original XA? If same, why it appear here?

both of them are distribution transaction. however

`XA` could do it automatically by using `begin`, `commit`/`rollback`
while `MANUALXA` is controlled by user themself, i.e. triggered by `XA` statements.